src/extensions/step_handlers/script_handler/src/script_handler.cpp

```diff
@@ -535,6 +536,14 @@ ScriptHandler_PerformAction(const std::string& action, const tagADUC_WorkflowDat
         return results;
     }
 
+    const ADUC_ConfigInfo* config = ADUC_ConfigInfo_GetInstance();
+    if (config == nullptr)
+    {
+        Log_Error("Failed to get config info instance");
```

Comment:
omit log to avoid adding string to the binary for something that will almost never happen

Reply:
Removed